### DISCUSSION TITLE
Error on upgrade typer

### DISCUSSION THREAD

**Discussion by gablucas** [oobabooga/text-generation-webui#5984] [General]
Hello friends, after installing oobabooga I went to install TTS

pip install --upgrade tts

But I received the error

_BUG: pip's dependency resolver currently does not take into account all packages that are installed. This behavior is the source of the following dependency conflicts.
fastapi-cli 0.0.2 requires typer>=0.12.3, but you have typer 0.9.4 which is incompatible._

So I updated typer, but I got another error

_BUG: pip's dependency resolver currently does not take into account all packages that are installed. This behavior is the source of the following dependency conflicts.
spacy 3.7.4 requires typer<0.10.0,>=0.3.0, but you have typer 0.12.3 which is incompatible.
weasel 0.3.4 requires typer<0.10.0,>=0.3.0, but you have typer 0.12.3 which is incompatible._

How can I resolve this?

**Comment by kissecx** [comment]
same here.

**Comment by UrfanAlvany** [comment]
did you found a solution for this ?

**Comment by acosme** [comment]
> Hello friends, after installing oobabooga I went to install TTS
> 
> pip install --upgrade tts
> 
> But I received the error
> 
> _BUG: pip's dependency resolver currently does not take into account all packages that are installed. This behavior is the source of the following dependency conflicts. fastapi-cli 0.0.2 requires typer>=0.12.3, but you have typer 0.9.4 which is incompatible._
> 
> So I updated typer, but I got another error
> 
> _BUG: pip's dependency resolver currently does not take into account all packages that are installed. This behavior is the source of the following dependency conflicts. spacy 3.7.4 requires typer<0.10.0,>=0.3.0, but you have typer 0.12.3 which is incompatible. weasel 0.3.4 requires typer<0.10.0,>=0.3.0, but you have typer 0.12.3 which is incompatible._
> 
> How can I resolve this?

Hi guys,

After some hours just solved doing this:

1 - Run: `./update_wizard_macos.sh`;
1.1- Choose: "B) Install/update extensions requirements" and after choose: "D) coqui_tts";
2 - Run `./cmd_macos.sh`;
2.1 - Inside the new subprompt run: `pip install --upgrade tts`;

In the last step it will download kind of 1.8GB data and then say it's all done and ask you to prompt "http://127.0.0.1:7860/".

Hope it helps!

(Here: Apple M2, MacB. Air)

**Reply by subri842** [reply]
I wasn't able to. How do I uninstall/remove oobabooga?
